terra_layer/style.py

```diff
@@ -455,31 +554,58 @@ def generate_style_from_wizard(layer, config):
                 'With "graduated" symbology, "boundaries" or "method" should be provided'
             )
 
+        include_no_value = config.get("include_no_value", True)
         if boundaries:
+            field_style = get_field_style(field)
+            no_value_fill_color = config.get(
```

Comment:
default_style should be a dict that you can merge with default values.

Reply:
Ok, use a dict with default values.